source/Tests/Converters/BlockOclConverterFixture.cs

```diff
@@ -50,6 +50,7 @@ public void IndexersAreIgnored()
 
         class WithIndexer
         {
+            public string Test { get; } = "Value";
```

Comment:
looks like this will either need to be split into two test classes or the asserts will need updating in order to pass the build.